### PR TITLE
[Feature] ImageNormalize

### PR DESCRIPTION
**Pull request by Yshuo-Li**
Mean Shift Layer

### CODE REVIEW

File: mmedit/models/components/layers/mean_shift.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import torch
+import torch.nn as nn
+
+from mmedit.models.registry import COMPONENTS
+
+
+@COMPONENTS.register_module()
+class MeanShift(nn.Conv2d):
+    """Mean shift.
```
Comment (innerlee) [inline mmedit/models/components/layers/mean_shift.py:9]: This is too sloppy

File: mmedit/models/components/layers/mean_shift.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+    Args:
+        pixel_range (float): Pixel range of image.
+        rgb_mean (Tuple[float]): Image mean in RGB orders.
+        rgb_std (Tuple[float]): Image std in RGB orders.
+        sign (int): Sign of bias. Default -1.
+    """
+
+    def __init__(self, pixel_range, rgb_mean, rgb_std, sign=-1):
+        super(MeanShift, self).__init__(3, 3, kernel_size=1)
```
Comment (innerlee) [inline mmedit/models/components/layers/mean_shift.py:19]: `super().__init__(...)`

File: mmedit/models/components/layers/mean_shift.py [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+        std = torch.Tensor(rgb_std)
```
Comment (innerlee) [inline mmedit/models/components/layers/mean_shift.py:20]: What's the device of this tensor

File: mmedit/models/components/layers/mean_shift.py [hunk 4]
```diff
@@ -0,0 +1,27 @@
+import torch
+import torch.nn as nn
+
+from mmedit.models.registry import COMPONENTS
+
+
+@COMPONENTS.register_module()
+class MeanShift(nn.Conv2d):
```
Comment (innerlee) [inline mmedit/models/components/layers/mean_shift.py:8]: MeanShift is a classical alogrithm, so I do not like this name. Do you have alternative choices? Like `FeatureShift` or so?

File: mmedit/models/components/layers/mean_shift.py (hunk 3 above)
Comment (Yshuo-Li) [reply mmedit/models/components/layers/mean_shift.py:20]: CPU.
It is not important, only used in init.

Comment (innerlee) [reply mmedit/models/components/layers/mean_shift.py]: No this should be taken care, becuase it might be run on gpu

Comment (Yshuo-Li) [reply mmedit/models/components/layers/mean_shift.py]: How about `del std` after __init__?

File: mmedit/models/components/layers/feature_shift.py [hunk 5]
```diff
@@ -0,0 +1,31 @@
+import torch
+import torch.nn as nn
+
+from mmedit.models.registry import COMPONENTS
+
+
+@COMPONENTS.register_module()
+class FeatureShift(nn.Conv2d):
```
Comment (innerlee) [inline mmedit/models/components/layers/feature_shift.py:8]: As discussed in private, it is a feature normalization. So "FeatureNormalize" maybe more suitable.

Comment (innerlee) [reply mmedit/models/components/layers/feature_shift.py:8]: ImageNormalize

File: mmedit/models/components/layers/mean_shift.py (hunk 3 above)
Comment (innerlee) [reply mmedit/models/components/layers/mean_shift.py]: My understanding was wrong. No need to change :)

**Review by innerlee — APPROVED** [review]
